deploy/base/crd.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
In regards to generating, I think it would be good to do this from the beginning to make sure we stay in sync. This is fairly straightforward with `controller-tools`. Here is an example of doing so with go generate --> https://github.com/crossplane/crossplane/blob/3ea5d036778e0c257960983d0cdb9271f12dc895/apis/generate.go#L26